modules/data-entry/src/main/frontend/src/dataHomepage/FormView.jsx

```diff
@@ -38,11 +38,16 @@ import { Link } from 'react-router-dom';
 import DescriptionIcon from '@material-ui/icons/Description';
 import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
 import DeleteButton from "./DeleteButton.jsx";
+import NewFormDialog from "./NewFormDialog.jsx";
 import { getEntityIdentifier } from "../themePage/EntityIdentifier.jsx";
 
 function FormView(props) {
-  const { classes } = props;
+  const { questionnaire, classes } = props;
   const [ activeTab, setActiveTab ] = useState(0);
+  const [ title, setTitle ] = useState(typeof(props.title) != 'undefined' ? props.title : "Forms");
+  const [ subtitle, setSubtitle ] = useState();
+  const [ questionnairePath, SetQuestionnairePath ] = useState();
```

Comment:
```suggestion
  const [ questionnairePath, setQuestionnairePath ] = useState();
```
To match the call on line 91